recipes/bdw-gc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "7.4.2" %}
+
+package:
+  name: bdw-gc
+  version: {{ version }}
+
+source:
+  fn: gc7_4_2.tar.gz
+  url: https://github.com/ivmai/bdwgc/archive/gc7_4_2.tar.gz
+  sha1: 441777b9865a730c9c804f5f5de2aa9a55fc9419
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - libatomic_ops
+
+test:
+  commands:
+    - true
```

Comment:
Let's try to avoid this. Is there a command to test? A library to check for existence? (Or both?)